### PR TITLE
[Home][ArtworkRail] Set a max of 99 artwork rails

### LINKED ISSUE #274: [Home] Include more generic gene rails
See https://artsy.wake.io/1085

### PR DESCRIPTION
**Pull request by sarahscott**
Does it actually have to be 999 or is 50 ok for now 😄 ?

Related to https://github.com/artsy/metaphysics/issues/410

Fixes #274

### CODE REVIEW

File: lib/containers/home.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default Relay.createContainer(Home, {
   fragments: {
     home: () => Relay.QL`
       fragment on HomePage {
-        artwork_modules {
+        artwork_modules(max_rails: 50) {
```
Comment (sarahscott) [inline lib/containers/home.js]: The current [max_rails](https://github.com/artsy/metaphysics/blob/master/schema/home/home_page_artwork_modules.js#L31) setup will grab more [generic genes](https://github.com/artsy/metaphysics/blob/master/schema/home/home_page_artwork_modules.js#L23) anyway, and [perhaps we'll make the api better in the future](https://github.com/artsy/metaphysics/issues/410).

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/d21cc1d7f6b206b5771eaaefe122672b39c4ef81/lib/containers/home.js">lib/containers/home.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by sarahscott** [comment]
Alles gut!

**Comment by alloy** [comment]
@sarahscott Actually two digits is fine, it saves us a byte on each request 😜 , but I think `99` feels more convention-y to signal “give me everything”.

**Comment by mennenia** [comment]
🎉

**Pull request merged**